proto-google-cloud-firestore-bundle-v1/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>proto-google-cloud-firestore-bundle-v1</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
If there isn't a strong reason to have this as 1.0.0 and can instead follow the other artifacts we can updated to the following to get version bumping automatically via release tooling.

```suggestion
    <version>1.0.0-SNAPSHOT</version>
    <version>2.0.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-firestore:current} -->
```

If we want to keep the 1.0.0 version, we'll need to add a new entry to `version.txt` in the root of the repo for the new artifact version, and then add the corresponding comment to get version bumping.

Reply:
Done.